batched/dense/impl/KokkosBatched_SVD_Serial_Impl.hpp

```diff
@@ -23,12 +23,14 @@
 namespace KokkosBatched {
 // Version which computes the full factorization
 template <typename AViewType, typename UViewType, typename VViewType,
-          typename SViewType, typename WViewType>
+          typename SViewType, typename WViewType, 
+          typename TolType>
 KOKKOS_INLINE_FUNCTION int SerialSVD::invoke(SVD_USV_Tag, const AViewType &A,
```

Comment:
This function is part of our public API, the change you are making will break the code of users who already use this function. A better change would be to introduce the new interface like so:

```c++
template <typename AViewType, typename UViewType, typename VViewType,
          typename SViewType, typename WViewType, 
          typename TolType>
KOKKOS_INLINE_FUNCTION int SerialSVD::invoke(SVD_USV_Tag, const AViewType &A,
                                             const UViewType &U,
                                             const SViewType &sigma,
                                             const VViewType &Vt,
                                             const WViewType &work,
                                             const TolType tol) {
...
}

template <typename AViewType, typename UViewType, typename VViewType,
          typename SViewType, typename WViewType>
KOKKOS_INLINE_FUNCTION int SerialSVD::invoke(SVD_USV_Tag, const AViewType &A,
                                             const UViewType &U,
                                             const SViewType &sigma,
                                             const VViewType &Vt,
                                             const WViewType &work) {
   SerialSVD::invoke(SVD_USV_Tag, A, U, sigma, Vt, work, some_default_tol);
}
```